src/emulator/gui/src/settings_dialog.cpp

```diff
@@ -75,12 +86,29 @@ void draw_settings_dialog(HostState &host) {
         ImGui::Checkbox("GUI Visible", &host.cfg.show_gui);
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Check the box to show GUI after booting a game.");
+        ImGui::SameLine();
+        ImGui::Checkbox("Game Background", &host.cfg.game_background);
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Uncheck the box to disable show Game background.");
         ImGui::Spacing();
         ImGui::SliderInt("Game Icon Size \nSelect your preferred icon size.", &host.cfg.icon_size, 16, 128);
         ImGui::Spacing();
-        ImGui::SliderFloat("Background Alpha \nSelect your preferred transparent background effect.", &host.cfg.background_alpha, 0.999f, 0.000f);
-        if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("Minimum slider is opaque and maximum is transparent.");
+        ImGui::PushItemWidth(400);
+        ImGui::InputTextWithHint("Set your Path for Background Image", "Write your path image here", &host.cfg.background_image);
+        ImGui::PopItemWidth();
+        if (ImGui::Button("Apply Change Image")) {
+            if (!host.gui.user_backgrounds[host.cfg.background_image]) {
+                init_background(host, host.cfg.background_image);
+            } else if (host.gui.user_backgrounds[host.cfg.background_image]) {
+                host.gui.current_background = host.gui.user_backgrounds[host.cfg.background_image];
+            }
+        }
+        ImGui::Spacing();
+        if (host.gui.current_background) {
+            ImGui::SliderFloat("Background Alpha \nSelect your preferred transparent background effect.", &host.cfg.background_alpha, 0.999f, 0.000f);
+            if (ImGui::IsItemHovered())
+                ImGui::SetTooltip("Minimum slider is opaque and maximum is transparent.");
+        }
```

Comment:
There are some weird grammar issues with some of the tool tips. Don't be afraid to use a tool like Grammarly or the Internet! I do it all the time.

Reply:
@Zangetsu Good job, man! Background of the game is working now, I still can not load a custom image, there must be something wrong here in my setup ... but this is already very good, thank you guys.

![Capturar2](https://user-images.githubusercontent.com/26087621/57488995-23e8f100-728b-11e9-919e-0af23d1070da.JPG)
![Capturar1](https://user-images.githubusercontent.com/26087621/57489008-29ded200-728b-11e9-863f-603faed87aa7.JPG)

Reply:
i have fixed normally